runtime-config-linux.md

```diff
@@ -144,6 +144,12 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+There are a few facets to this:
+
+* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
+* Assigning processes to cgroups.
+* Configuring cgroups to apply resource limits (`resources`).
+
```

Comment:
On Wed, Nov 18, 2015 at 03:35:28AM -0800, Qiang Huang wrote:
> > +There are a few facets to this:
> > +
> > +* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
> > +* Assigning processes to cgroups.
> > +* Configuring cgroups to apply resource limits (`resources`).
> 
> I don't know, if it's general facets for cgroup, I don't think it's
> typical enough and helps much.  I suggest we remove this hunk, the
> others look good to me.

I think the lack of a distinction between “what the cgroup is called”
and “how the cgroup is configured” is part of the confusion we've seen
around ‘cgroupsPath’ and ‘resources’.  I was trying to clarify that
distinction here, but if it doesn't need to be clarified (or there's a
better way to clarify it), I'm happy drop (or replace) this hunk.